### ISSUE TITLE
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, at Home screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#866]
**Describe the bug**
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, at Home screen.

**To Reproduce**
1. Go to Home screen (http://localhost:9992/cm/app/?view=home&pathType=unknown)
2. Click on 'Browse Library' button.
3. Now, while browsing 'Sites' or 'Assets' or 'Design' or 'Recycling', if you mouse over them, You will be able to see their title but when you hover over that title, it disappears.

**Expected behavior**
User should be able to move pointer over the additional content (tool tip, label etc.), and this additional content should not disappear, while moving focus on it.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by rgarg24** [comment]
There is no additional content but title. So, closing this bug.
